src/main/groovy/org/gradle/android/AndroidCacheFixPlugin.groovy

```diff
@@ -55,14 +55,10 @@ class AndroidCacheFixPlugin implements Plugin<Project> {
 
     @Override
     void apply(Project project) {
-        workarounds.addAll(initializeWorkarounds(project))
+        workarounds.addAll(initializeWorkarounds())
 
-        if (!isSupportedAndroidVersion(project)) {
-            if (isMaybeSupportedAndroidVersion(project)) {
-                Warnings.MAYBE_SUPPORTED_ANDROID_VERSION.warnOnce(project)
-            } else {
-                throw new RuntimeException("Android plugin ${CURRENT_ANDROID_VERSION} is not supported by Android cache fix plugin. Supported Android plugin versions: ${SUPPORTED_ANDROID_VERSIONS.join(", ")}. Please check if a newer version of this plugin is available or override with -D${IGNORE_VERSION_CHECK_PROPERTY}=true.")
-            }
+        if (!isMaybeSupportedAndroidVersion()) {
+            throw new RuntimeException("Android plugin ${CURRENT_ANDROID_VERSION} is not supported by Android cache fix plugin. Supported Android plugin versions: ${SUPPORTED_ANDROID_VERSIONS.join(", ")}")
```

Comment:
Once we resolve the discussion on what max/min ranges are for this, I think we should update this error message to be more precise.

Reply:
discussion on max/min resolved, what are your suggestions for this message?

Reply:
In this case this means that the AGP version is too old. In that case I think we should recommend that they use the Android Cache Fix plugin which is compatible with their version. What do you think about: `Android plugin ${CURRENT_ANDROID_VERSION} is not supported by Android cache fix plugin. For older Android Gradle Plugin versions, please use Android Cache Fix Plugin 2.4.6`

Reply:
done